### ISSUE TITLE
Rendering property with allOf ignores title and description

### ISSUE THREAD

**Issue by adam-homeboost** [Authress-Engineering/openapi-explorer#257]
I have an object with an `allOf` listing a $ref.  In this case, the `title` and `description` fields are ignored and only the schema name is used in those locations. 
```
{
  "openapi": "3.1.0",
  "info": {
    "title": "Some API",
    "summary": "",
    "description": "",
    "version": "alpha"
  },
  "paths": {
    "/api/v1/whatever/": {
      "post": {
        "parameters": [],
        "responses": {
          "200": {
            "description": "A Response",
            "content": {
              "application/json": {
                "schema": {
                  "$ref": "#/components/schemas/Response"
                }
              }
            }
          }
        }
      }
    }
  },
  "components": {
    "schemas": {
      "NumericValue_str_": {
        "properties": {
          "nv": {
            "type": "string"
          }
        }
      },
      "Response": {
        "properties": {
          "value": {
            "allOf": [
              {
                "$ref": "#/components/schemas/NumericValue_str_"
              }
            ],
            "title": "My Title",
            "description": "My Description"
          }
        }
      }
    }
  }
}
```

Result:

![image](https://github.com/Authress-Engineering/openapi-explorer/assets/127809317/2c849aef-3d71-4845-be2c-d4e2e66cdabc)

Wanted:

![image](https://github.com/Authress-Engineering/openapi-explorer/assets/127809317/a6c58734-970c-41b9-991b-3d919b852292)

(also, I'm not clear on the what is happening with the font size there, but honoring the title and description content is what I am after here).

If the $ref is moved outside an `allOf` or if its an `anyOf` instead, then it works as expected.  Having more values in the `allOf` does not help.

**Comment by wparad** [comment]
Hmm, looking at what you have shown, it looks like the type column might be correct but the description is wrong. The type column is meant to represent the actual named typed that is created from SDK generators, etc.. not a helpful field name. Agreed though in the case of "allOf" it isn't clear what that table cell display string should actually be. For instance if you have an SDK for your API what is the name of the schema?

**Comment by adam-homeboost** [comment]
Yes, I agree that the type ought to remain as the actual schema name.  The description is what is needed more.  Cases like this, where there is ambiguity is where its most important to be able to manually control this area.  I will admit that I am unclear on just why my schema generator is choosing to use `allOf` here, especially with only one value, but so far as I can tell, it is a legitimate use case in general.  For this specific instance, I think my schema generator could do a better job at what I am asking it to do, but I don't think that problem is as relevant so long as I can get the overriding title and description to work.

**Comment by wparad** [comment]
Yeah, that part is definitely seems a bug. Thanks for finding this.

**Comment by wparad** [comment]
Fixed in `2.2.719`.
